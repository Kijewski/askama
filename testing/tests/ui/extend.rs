use askama::Template;

#[derive(Template)]
#[template(
    source = r##"bla
{% extends "base.html" %}
"##,
    ext = "txt",
    print = "ast"
)]
pub struct X;

fn main() {}
