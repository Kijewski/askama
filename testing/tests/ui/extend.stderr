error: `extends` block must come first in a template
 --> <source attribute>:2:2
       " extends \"base.html\" %}\n"
 --> tests/ui/extend.rs:5:14
  |
5 |       source = r##"bla
  |  ______________^
6 | | {% extends "base.html" %}
7 | | "##,
  | |___^
